apps/frontend/src/libs/components/sidebar/sidebar-item/styles.module.css

```diff
@@ -0,0 +1,28 @@
+.navigation-link {
+	display: flex;
+	flex-direction: row;
+	gap: 8px;
+	align-items: center;
+	padding: 12px 16px;
+	color: var(--color-text-secondary);
+	text-decoration: none;
+}
+
+.navigation-link:hover {
+	color: var(--color-text-primary);
+	background-color: var(--color-background-secondary);
+	border-radius: 6px;
+}
+
+.navigation-link:hover svg {
+	color: var(--color-brand-primary);
+}
+
+.navigation-item-label {
+	font-size: 16px;
+	line-height: 120%;
+}
+
+.navigation-link:hover .navigation-item-label {
```

Comment:
The navigation-link itself should have cursor pointer, not only label

Reply:
The problem is that browser set cursor: default to label and cursor: pointer to link. So if i delete it and put cursor: pointer to navigation-link nothing changes, but when hover over text cursor won`t be pointer